tough/src/schema/mod.rs

```diff
@@ -292,7 +292,7 @@ pub struct Snapshot {
 ///    },
 /// ```
 #[derive(Debug, Clone, Deserialize, Serialize, Eq, PartialEq)]
-pub struct SnapshotMeta {
+pub struct Metafiles {
```

Comment:
I think this should be `Metafile` since there's only one. I was thrown at first by the `Hashes` struct immediately after this one, but that's different because it can include multiple hashes, even though only the sha256 one is used.

In the enclosing structs, the `HashMap<String, Metafile>` collection implies the "plural" - there can be 0-N metafiles - so the type name doesn't need to reflect it. If you really wanted a closer correspondence to the spec, you could declare a newtype:
```
struct Metafiles(HashMap<String, Metafile>)
```

And then `Snapshot` and `Timestamp` could say this instead:
```
    pub meta: Metafiles,
```

But I don't think that's necessary.

Reply:
@bcressey: sorry, should I go ahead and rename `Metafiles` to `Metafile`?

If you want I can also go ahead with the `struct Metafiles` thing you suggested.

Reply:
> sorry, should I go ahead and rename Metafiles to Metafile?

@flavio: yes please rename `Metafiles` to `Metafile`. I wouldn't bother with the second part (newtype) unless you're excited about it 😀

Reply:
I’m currently on vacation. I’ll look at that on the 1st of August

Reply:
I'm back from holidays. I've done the rename inside of 0274aa528f7f3101ebcf5c7b69971e9522f20be3

I tried to introduce the `struct Metafiles`, but the change required to implement `get` and `insert` methods on it. I don't know if we really want all this extra code.

I'll leave you to mark this comment as solved if you're fine with the change